communication/calendar-guidelines.md

```diff
@@ -95,6 +100,54 @@ will prevent you from getting updates like a canceled meeting. Join the main
 
 Accept the invite from the sender and you'll have the updates.
 
+### Calendar event template
+
+In case you are creating a calendar event and wondering what to include in it, you can refer to the following template
+
+#### Title
+
+```
+[SIG-<name>] <meeting title>
+or
+[WG-<name>] <meeting title>
+or
+[User Group-<name>]
+```
+#### Description
+
+```
+<One line intro about the SIG/ WG. Can include github links>
+
+Time: <Weekly/ biweekly/... or is the meeting for a fixed time and till when?>
+
+Meeting Notes & Agenda: <Hyperlink to the meeting notes doc>
+
+Zoom link: <Zoom invite link. Preferably both app and web client links>
+
+Additional Info: <Any extra info/ links can be provided here>
+
+Livestream link: <optional>
+
+Youtube Playlist: <optional>
+
+Google group: <optional>
+```
+
+#### Example
+
+> SIG contribex sub project, Contributor Marketing team weekly discussion
+> 
+> https://github.com/kubernetes/community/tree/master/communication/marketing-team
+> 
+> **Time**: Weekly. Every Friday
+> 
+> **Meeting Notes & Agenda**: https://docs.google.com/document/d/1IlHAJ131akGhI5ffF4OoVW0PrsVY4C0BB8l-UyQaQVo/edit
+> 
+> **Zoom link**:
+>
+> &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; App: https://zoom.us/j/596959769
+>
```

Comment:
```suggestion
> One tap mobile
+16465588656,,596959769# US (New York)
+16699006833,,596959769# US (San Jose)

Dial by your location
        +1 646 558 8656 US (New York)
        +1 669 900 6833 US (San Jose)
Meeting ID: 596 959 769
Find your local number: https://zoom.us/u/acQiheRjN
```

Reply:
If this is going to be included, I'd follow the same suggestion as above and replace the number with `XXX XXX XXX` 👍

Reply:
:+1:

Reply:
Considering there are different numbers for different locations, should we include mobile info? 
Also adding more info, might make it clumsy. Like say when you hover over an event on the calendar. What do you say?

Reply:
Good call @rajula96reddy, +1 to skipping it. It'd be good to remind people to include a #, but better not to list them.